### DISCUSSION TITLE
Use Spark load to import data to check the task execution status, but show progress exceptions

### DISCUSSION THREAD

**Discussion by Amanwan** [apache/doris#29961] [A - General / Q&A]
doris版本:2.0.3
Spark Load时查看任务执行状态进度异常
![image](https://github.com/apache/doris/assets/57030271/611ef05b-d736-4db6-a957-8a687a004b90)

同时在 /spark_launcher_log 目录下对应的日志也不全，具体如下图
![image](https://github.com/apache/doris/assets/57030271/860f3389-1b49-4ca4-900a-1a2e679366ec)

有大神知道这是什么原因吗

**Comment by yagagagaga** [comment]
推荐使用Spark Doris connector或者Hive Catalog导入数据

**Reply by Amanwan** [reply]
相比较于spark load  使用Spark Doris connecto有什么优势嘛
